aea/configurations/base.py

```diff
@@ -617,8 +618,8 @@ def __init__(
         self,
         name: str,
         author: str,
```

Comment:
Name and author are required arguments for each config object. We had implied this but never actually enforced it.